### PR TITLE
Created Hello Bot Workshop

### PR DESCRIPTION
**Pull request by hickorysb**
The Hello Bot workshop is about creating a basic Discord bot with a hello world command.

### CODE REVIEW

**Comment by zachlatta** [comment]
Hell yeah! Thanks for submitting excited to hear what the team thinks when they review

--
Zach Latta
Founder, Hack Club | zach@hackclub.com | Donate ( https://hackclub.com/donate )
@zachlatta ( https://twitter.com/zachlatta ) / fb ( https://facebook.com/crynix ) / web ( https://zachlatta.com ) / github ( https://github.com/zachlatta )

**Comment by polytroper** [comment]
Hey @hickorysb thanks for this! Would love to have this workshop on the site. A few change requests:

1. Can you add some more whitespace? I think the code blocks will break things up a bit when it's rendered, but right now this seems kind of hard to read.
2. The metadata at the top of the file seems to be formatted incorrectly. Check out the other workshops to see the format.
3. Could you change the beginning of the workshop to use repl.it instead of local npm in the terminal? It's much easier to set up, the bot is automatically deployed, and it works on a chromebook (which is all that many students have to work on)

**Comment by maxwofford** [comment]
@polytroper see latest changes on GitHub https://github.com/hackclub/hackclub/pull/1247/files

**Comment by polytroper** [comment]
@maxwofford Oh cool, that looks much better. I still think this should be done on repl.it though, as many students cannot locally install npm. It also would dramatically simplify the setup for this workshop, and keep the bot alive and hosted on repl.it. @hickorysb thoughts?

**Comment by hickorysb** [comment]
I will write the change to put it on repl.it. It will take me a little while though.

**Comment by hickorysb** [comment]
@polytroper It's been updated to Repl.it when you get a chance to review it.

File: workshops/hello_bot/README.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+This code will now only print the command from the message. Now we should actually have the bot send a message in Discord! For this workshop, the command will be `hello` and the response from the bot will be `world`. We can do that by updating the listener to the following:
+```js
+bot.on("message", (message) => {
+  if(!message.content.startsWith(prefix)) return;
+  var command = message.content.slice(prefix.length);
+  if(command == "hello") {
+    message.reply("world");
+  }
+});
+```
```
Comment (polytroper) [inline workshops/hello_bot/README.md]: Include the full source code at the end, so people can check their code against yours. Something like "Your whole file should now look like this: <code>"

**Comment by polytroper** [comment]
LGTM. Thanks! :+1:

**Pull request merged**